sycl/source/detail/scheduler/scheduler.cpp

```diff
@@ -73,6 +73,9 @@ EventImplPtr Scheduler::addCG(std::unique_ptr<detail::CG> CommandGroup,
       NewCmd = MGraphBuilder.addCGUpdateHost(std::move(CommandGroup),
                                              DefaultHostQueue);
       break;
+    case CG::HOST_TASK_CODEPLAY:
```

Comment:
Minor: should it be `CODEPLAY_HOST_TASK` to keep it consistent with the API name?

Reply:
It's consistent with another enum's value name.

Reply:
"another enum's value name" is not consistent with API as well. Please, fix both.

Reply:
Done.